library/jetty

```diff
@@ -287,27 +287,42 @@ GitCommit: c0f30f8462d083ddc44b6af8ae967315421b6451
 Tags: 9.4.56-jdk11-amazoncorretto, 9.4-jdk11-amazoncorretto, 9-jdk11-amazoncorretto
 Architectures: amd64, arm64v8
 Directory: amazoncorretto/9.4/jdk11
-GitCommit: c0f30f8462d083ddc44b6af8ae967315421b6451
+GitCommit: 09303c6a1c73276d5cd286dbd5f04a1a752ecf62
 
-Tags: 12.0.14-jdk21-alpine-amazoncorretto, 12.0-jdk21-alpine-amazoncorretto, 12-jdk21-alpine-amazoncorretto
+Tags: 12.0.15-jdk23-al2023-amazoncorretto, 12.0-jdk23-al2023-amazoncorretto, 12-jdk23-al2023-amazoncorretto
+Architectures: amd64, arm64v8
+Directory: amazoncorretto/12.0/jdk23-al2023
+GitCommit: 09303c6a1c73276d5cd286dbd5f04a1a752ecf62
+
+Tags: 12.0.15-jdk21-alpine-amazoncorretto, 12.0-jdk21-alpine-amazoncorretto, 12-jdk21-alpine-amazoncorretto
 Architectures: amd64, arm64v8
 Directory: amazoncorretto/12.0/jdk21-alpine
-GitCommit: f0c6a3e167722283788502f39ee521ffdac911a0
+GitCommit: 09303c6a1c73276d5cd286dbd5f04a1a752ecf62
+
+Tags: 12.0.15-jdk21-al2023-amazoncorretto, 12.0-jdk21-al2023-amazoncorretto, 12-jdk21-al2023-amazoncorretto
+Architectures: amd64, arm64v8
+Directory: amazoncorretto/12.0/jdk21-al2023
+GitCommit: 09303c6a1c73276d5cd286dbd5f04a1a752ecf62
 
-Tags: 12.0.14-amazoncorretto, 12.0-amazoncorretto, 12-amazoncorretto, 12.0.14-jdk21-amazoncorretto, 12.0-jdk21-amazoncorretto, 12-jdk21-amazoncorretto
+Tags: 12.0.15-amazoncorretto, 12.0-amazoncorretto, 12-amazoncorretto, 12.0.15-jdk21-amazoncorretto, 12.0-jdk21-amazoncorretto, 12-jdk21-amazoncorretto
```

Comment:
Should the non-alpine amazoncorretto images have extra tags added with `-al2` in them so that the default could move to `-al2023` at some future date?

Reply:
Good idea, I will add this on the next PR.